test/helper.rb

```diff
@@ -5,7 +5,7 @@
 
   SimpleCov.start do
     enable_coverage :branch
-    add_filter "/test/"
+    add_filter{|f| f.filename.match?(%r{\A#{Regexp.escape(__dir__)}/})}
```

Comment:
```suggestion
    add_filter{|f| f.filename.start_with?(__dir__)}
```

Is this easier to read? Otherwise, I don't have much opinion.

Reply:
it misses the following slash.  It's not a big deal, but if we ever added something that started with `test`, it would filter that out too.

I guess we could do:

`add_filter{|f| f.filename.start_with?(__dir__ + "/")}`